zokrates_core_test/tests/tests/bool_compare.zok

```diff
@@ -0,0 +1,3 @@
+def main(field a, field b) -> (field):
```

Comment:
Would it make sense to pass booleans as inputs and return booleans? It would make the code clearer